### PR TITLE
[doc] supabase documentation

### PR DESCRIPTION
**Pull request by jinkang-0**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

### :palm_tree: What's new in this PR
[//]: # "REQUIRED - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

Adds documentation to explain how we use Supabase

## :deciduous_tree: How to review
[//]: # 'REQUIRED - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

Checkout the branch, install dependencies using `npm ci`, then run `npm docs:dev` to launch a server for `localhost:8080`.

[//]: # 'This tags the project leader as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
CC: @varortz

### CODE REVIEW

File: docs/dev/app/supabase.md [hunk 1]
```diff
@@ -2,4 +2,51 @@
 next: /dev/retool
 ---
 
+[[toc]]
```
Comment (varortz) [inline docs/dev/app/supabase.md]: Could we remove the `[[toc]]`? It seems a bit redundant with the "On This Page" dropdown:
image

Or is there a way we can remove the dropdown? 🤔

Comment (jinkang-0) [reply docs/dev/app/supabase.md]: I'll remove the `[[toc]]`

**Review by varortz — APPROVED** [review]
lgtm! Looks good to merge after addressing the comment below.

File: docs/dev/app/supabase.md [hunk 2]
```diff
@@ -3,3 +3,48 @@ next: /dev/retool
...
+
+This function fetches all cases, each joined with the corresponding language requirements and relief codes sought, both aggregated in an array of their own.
+
+`get_document_translations`
+
+This function fetches all document translation listings, each joined with the corresponding language requirements aggregated in an array.
+
+`get_interpretations`
+
+This function fetches all one-off interpretation listings, each joined with the corresponding language requirements aggregated in an array.
```
Comment (varortz) [inline docs/dev/app/supabase.md]: Can we update "one-off" to "one-time"?

**Pull request merged**